buildroot-external/patches/occu/0039-WebUI-Fix-Style-Glitches/occu/WebUI/www/webui/style.css

```diff
@@ -2293,6 +2293,14 @@ table.center {
   font-weight: bold;
 }
 
+.controlHeader150px {
```

Comment:
What purpose does this change serve here? Can't see `controlHeader150px` be used anywhere?!?

Reply:
I don't know. It's part of the original `style.css` from occu. See here https://github.com/eq-3/occu/blob/70db0f403ca72d235ce5cf6416b6534daac83f95/WebUI/www/webui/style.css#L2264